language-bindings/rust/build.rs

```diff
@@ -0,0 +1,30 @@
+extern crate bindgen;
```

Comment:
Should add the license header, note that the license must be "Apache-2.0 WITH LLVM-exception", refer to:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/libraries/lib-wasi-threads/lib_wasi_threads_wrapper.c#L2-L3
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/libraries/debug-engine/debug_engine.c#L2-L3

Reply:
Sure, makes sense. Does the license header look correct now?
I also added the license to the `Cargo.toml`.

Reply:
Yes, it looks good. BTW, I didn't find license header in Cargo.toml of this folder (it is added the file of another folder), could you add it?

Reply:
Oh, I meant I added the license information field:
```toml
  license = "Apache-2.0 WITH LLVM-exception"
```

But I can also leave a license header there 😊

Reply:
Oh, right, my bad. There are two `Cargo.toml` and now they should  both have a License header and a license field for the crate:
```toml
  license = "Apache-2.0 WITH LLVM-exception"
```